build/common.gni

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains common build flags for building Crosswalk. Users should
+# NOT directly import this file into their build folder.
+# See //xwalk/build/linux.gni for how to import flags for Crosswalk-Linux Build
+
+root_extra_deps = [ "//xwalk:xwalk_builder" ]
+
+# From src/build/config/features.gni
+# TODO: Enable nacl for Linux.
```

Comment:
It is done now.  Previously I found even in gyp it cannot build with `disable_nacl = 0`,  In lastest code everything works fine.